### PR TITLE
fix repo fingerprints

### PR DESCRIPTION
**Pull request by ysyoung**

### CODE REVIEW

**Comment by nhandler** [comment]
Ship It!
Ship It!_at_the_San_Diego_Bay_04.jpg)

File: yelp_package/dockerfiles/itest/marathon/Dockerfile [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 FROM ubuntu:trusty
 RUN echo "deb http://repos.mesosphere.com/ubuntu trusty main" > /etc/apt/sources.list.d/mesosphere.list
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv E56151BF
```
Comment (solarkennedy) [inline yelp_package/dockerfiles/itest/marathon/Dockerfile]: Can you use this long fingerprint in the other dockerfiles as well?

```
$ git grep E56151BF
yelp_package/dockerfiles/itest/chronos/Dockerfile:RUN apt-key adv --keyserver keyserver.ubuntu.com --recv 81026D0004C44CF7EF55ADF8DF7D54CBE56151BF
yelp_package/dockerfiles/itest/marathon/Dockerfile:RUN apt-key adv --keyserver keyserver.ubuntu.com --recv 81026D0004C44CF7EF55ADF8DF7D54CBE56151BF
yelp_package/dockerfiles/itest/mesos/Dockerfile:RUN apt-key adv --keyserver keyserver.ubuntu.com --recv 81026D0004C44CF7EF55ADF8DF7D54CBE56151BF
```

Comment (solarkennedy) [inline yelp_package/dockerfiles/itest/marathon/Dockerfile]: Oh, nm already are of course :)

**Comment by solarkennedy** [comment]
ship

**Pull request merged**